CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.6.2] - 2017-04-12
+
+- Updates `clj-kitchensink' to 2.3.0
+
```

Comment:
Looks like we had a couple of other changes since 0.6.1 that we should include in the CHANGELOG.md:

- added the test jar for rbac-client
- bumped snakeyaml from 1.13 to 1.18

Also, I think it would be good to do a Y bump (to 0.7.0) for the snakeyaml bump since it technically has some breaking changes at the Java layer in it, although they are minor.  See conversation on #50.